### DISCUSSION TITLE
Dark mode README banner.

### DISCUSSION THREAD

**Discussion by macuyler** [polybar/polybar#2706] [Ideas]
GitHub recently added support for showing different images depending on the user's theme. It would be cool if there was a banner with white text in the README for dark mode users. [GitHub Docs: Specifying the theme an image is shown to](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)

My image editing skills end at running a python script to crudely mask the colors. But I added a proof of concept image to the [README](https://github.com/macuyler/polybar/blob/white-banner/README.md) on my fork to show it working.

Maybe @tobaloidee would be willing to take a shot at making an official dark mode banner?
Also, I'm not sure if the icon would look better with a white or black foreground in dark mode.

![Screen Shot 2022-05-01 at 2 45 22 PM](https://user-images.githubusercontent.com/10450774/166162661-5576002f-fe4c-478c-9104-5f2ed6f3bc7e.png)
![Screen Shot 2022-05-01 at 2 45 25 PM](https://user-images.githubusercontent.com/10450774/166162666-d9924282-69b1-4057-8a4d-3b0ba95389c2.png)

**Comment by patrick96** [comment]
I player around a bit with the original vector files @Tobaloidee provided and I came up with two other variants:

![2022-05-01-233016_1244x899_scrot](https://user-images.githubusercontent.com/2452038/166165338-7c1618bb-132b-4b92-ab4d-4f3a6c3af5d8.png)
![2022-05-01-233018_1244x899_scrot](https://user-images.githubusercontent.com/2452038/166165339-b599c299-33a0-4d35-ab50-ff58710edc54.png)

The first one is one of the original designs and the second one is just the regular banner with white text.

Personally I like the second one better. It also keeps the same logo in both modes and just makes the text better readable. What do you think?

**Reply by macuyler** [reply]
Yeah, I think the second one with the orange logo looks great!

**Reply by macuyler** [reply]
Maybe doing "Polybar" in orange, with the same orange logo would look good too? Instead of adding white as a third color.

**Reply by patrick96** [reply]
That's how that would look:

![2022-05-04-210358_1244x899_scrot](https://user-images.githubusercontent.com/2452038/166807470-08ebd6d3-5e57-4ea7-ab2f-ac9eb374a32b.png)

I like the white text better, I'll open a PR for that.
